packages/launchpad/src/components/code/FileRow.vue

```diff
@@ -96,12 +96,11 @@ const props = defineProps<{
   filePath: string
   content: string
   description?: string
+  fileExtension: string
 }>()
 
-// TODO: Remove this. Use FileParts when available
 const language = computed(() => {
-  // get the extension of the current file path
-  const extension = /\.(\w+)$/.exec(props.filePath)?.[1]
+  let extension = props.fileExtension.replace(/^\./, '')
```

Comment:
Getting the extension from gql now, but it comes prepended with a period, which we need to strip to validate against our supported languages.

Reply:
Agreed, here's the implementation for reference: https://github.com/cypress-io/cypress/blob/92eac2f67e4a0391bf058f8f95b64cc731d609cf/packages/graphql/src/schemaTypes/objectTypes/gql-FileParts.ts#L46-L47